terraform/gce-with-container/network.tf

```diff
@@ -80,6 +80,19 @@ resource "google_compute_firewall" "allow_tag_prysm_rpc" {
   }
 }
 
+resource "google_compute_firewall" "allow_tag_prysm_json_rpc" {
+  count         = var.create_firewall_rule ? 1 : 0
+  name          = "${var.prefix}-${local.instance_name}-ingress-tag-prysm-json_rpc-${var.environment}"
+  description   = "Ingress to allow beacon node json RPC ports to machines with the 'prysm-rpc' tag"
+  network       = var.network_name
+  source_ranges = ["0.0.0.0/0"]
+  target_tags   = ["prysm-rpc"]
+  allow {
+    protocol = "tcp"
+    ports    = [var.prysm_json_rpc_port]
```

Comment:
We could also reuse the `allow_tag_prysm_rpc` firewall rule above  and simply add the `var.prysm_json_rpc_port` to the list since they're both sharing the same target tag.
Either way works for me